docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
   # officially supported).
   # To run in macOS M1, set FLEET_MYSQL_IMAGE=arm64v8/mysql:oracle FLEET_MYSQL_PLATFORM=linux/arm64/v8
   mysql:
-    image: ${FLEET_MYSQL_IMAGE:-mysql:5.7.21}
+    image: ${FLEET_MYSQL_IMAGE:-mysql:8.0}
```

Comment:
What we run on our hosts should match what runs on CI (`8.0.36`)

Reply:
Done